### DISCUSSION TITLE
Php version

### DISCUSSION THREAD

**Discussion by ghost** [SwiftyEdit/SwiftyEdit#42] [General]
Hello, is it planned to put a development branch for php 8.2?
The current version in php 7.3 has been deprecated since November 2020 and is no longer maintained since November 2021.
See https://www.php.net/supported-versions.php

**Comment by patkon** [comment]
Hi @David-Baron 
I agree with you, and I think we should kick out PHP 7 as soon as possible. Maybe with the next release.
Are there that much difference between PHP 8.1 and PHP 8.2? My sites all run on 8.1.1
I will test 8.2 right away.

**Reply by ghost** [reply]
It works but with warnings and deprecations.

Example: in core/functions/func_basics line 170 parse_str the second param is mandatory in php 8 see: https://www.php.net/manual/en/function.parse-str.php

**Reply by patkon** [reply]
Thanks for testing. There are a lot of changes to be made.
